apps/dashboard/app/javascript/packs/config.js

```diff
@@ -38,11 +38,34 @@ function navbarHighlight(theme, color) {
     }`;
 }
 
-function getConfigData() {
+function maxFileSize () {
   const cfgData = configData();
 
-  return cfgData;
+  // Check if cfgData['maxFileSize'] is just empty string, 
+  // if so set default of maxFileUpload=10737420000 bytes.
+  if (cfgData['maxFileSize'].length == 0) {
+    return parseInt(10737420000, 10);
+  }
+  else {
+    const maxFileSize = cfgData['maxFileSize'];
+    return parseInt(maxFileSize, 10);
+  }
+}
+
+function transfersPath() {
+  const cfgData = configData();
+  const transfersPath = cfgData['transfersPath'];
+
+  return transfersPath;
+}
+
+function csrf_token() {
```

Comment:
Can we use camelCase for function names instead of snake_case.

Reply:
Yep! I only did it to make the change easier to push out, good call out.